corehq/apps/smsbillables/migrations/0019_deactivate_hardcoded_twilio_gateway_fees.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.12 on 2018-04-17 18:51
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from corehq.messaging.smsbackends.twilio.models import SQLTwilioBackend
+from corehq.sql_db.operations import HqRunPython
+
+
+def deactivate_hardcoded_twilio_gateway_fees(apps, schema_editor):
+    SmsGatewayFeeCriteria = apps.get_model('smsbillables', 'SmsGatewayFeeCriteria')
+
+    to_deactivate = SmsGatewayFeeCriteria.objects.filter(
+        backend_api_id=SQLTwilioBackend.get_api_id(),
+        country_code__isnull=False,
+        is_active=True,
+    )
+    remaining = SmsGatewayFeeCriteria.objects.filter(
+        backend_api_id=SQLTwilioBackend.get_api_id(),
+    ).exclude(
+        id__in=to_deactivate.values('id')
+    )
+    assert remaining.count() == 2
+    assert remaining.filter(country_code=None).count() == 2
+    to_deactivate.update(is_active=False)
+
+    assert SmsGatewayFeeCriteria.objects.filter(
+        backend_api_id=SQLTwilioBackend.get_api_id(),
+        is_active=True,
+    ).count() == 2
+    assert SmsGatewayFeeCriteria.objects.filter(
+        backend_api_id=SQLTwilioBackend.get_api_id(),
+        country_code__isnull=True,
+        is_active=True,
+    ).count() == 2
```

Comment:
how come these two are still needed if the smsbillables look up the fees afterwards?